### ISSUE TITLE
Training from scratch

### ISSUE THREAD

**Issue by 1349949** [TuSimple/LiDAR_RCNN#6]
Hi guys,
     It's really a nice work! 
     According to the paper and the code，training the pointnet is using the propsals generated by pre-trained detectors(such as pointpillar). 
     So the training of the base detectors and the pointnet is separated.
     I wonder whether you have tried to train the base detectors and the pointnet from scrath. That is, instead of  using pre-trained base detector to generate proposals，training the detector and the pointnet jointly from scrath. 
     If we train the network in this way，the results would be better，worse，or almost the same？

**Comment by Lzc6996** [comment]
@1349949 
The advantage of end-to-end training is that the features in the first stage can be jointly trained and reused. However, unlike Faster R-CNN, our model inputs the original point clouds of `[x, y, z]` which is the same as R-CNN. Therefore, there is no need for end-to-end training for LiDAR R-CNN. 
If you try it in an end-to-end manner, remember to add the feature from 1st stage as inputs to keep the gradient flow..I didn't try this setting, you can try it anyway.

**Comment by 1349949** [comment]
Thanks for your replying.
